doc/source/merge_changelogs.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Append release notes from other projects of the larray-project constellation to the current changelog
+"""
+
+import os
+import requests
+from subprocess import check_call
+
+
+# TODO: add Eurostat project
+EDITOR_GITHUB_REP = "https://github.com/larray-project/larray-editor"
+
+
+def short(rel_name):
+    return rel_name[:-2] if rel_name.endswith('.0') else rel_name
+
+
+def relname2fname(release_name):
+    short_version = short(release_name)
+    return r"version_{}.rst.inc".format(short_version.replace('.', '_'))
+
+
+def yes(msg, default='y'):
```

Comment:
Can't we reuse those 3 functions (and possibly others) from some common lib? (or officially moved them in larray "core" and make other projects use that here???). I never actually thought this through (!!!) so I don't know what is the best solution, but I always wanted and still would like to avoid duplicating code between the larray projects...